css-components/nav.css

```diff
@@ -1,8 +1,26 @@
 .nav {
+  width: 37.5rem;
+  height: 8.8rem;
+  background-color: var(--black);
+  list-style: none;
+  color: var(--light-grey);
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
 }
 
 .nav__button {
 }
 
 .nav__button--new-post {
+  width: 4rem;
+  height: 4rem;
+  background: var(--gradient);
+  border-radius: 50px;
+}
+
+.position_plus {
+  position: absolute;
```

Comment:
Position absolute can be deleted because of position fixed in the bottom line

Reply:
hatte position fixed auf der ganz flaschen klasse. ist gefixed und nun auf der .nav